### ISSUE TITLE
Message button in thread does not redirect to the discussion

### ISSUE THREAD

**Issue by singh-ravi-siso** [RocketChat/Rocket.Chat#20934]
### Description:
The message button(1 message) in the thread does nothing on click. Ideally, it should redirect to the respective discussion.
![Screenshot from 2021-02-28 16-53-20](https://user-images.githubusercontent.com/48203006/109416710-ec11e700-79e5-11eb-8f08-5a7a33a6a6e1.png)

### Steps to reproduce:

1. Create a discussion with a message
2. Open that thread
3. And click on the message button in the thread

### Expected behavior:

It should have redirected to the respective discussion.

### Actual behavior:

It does not redirect to the discussion.

### Server Setup Information:

- Version of Rocket.Chat Server: 3.11.1-develop
- Operating System: Ubuntu
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: v14.16.0
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version: Chrome
- Operating System: Ubuntu

### Additional context

**Working on it.**

**Comment by singh-ravi-siso** [comment]
The same bug persists in starred messages, pinned messages as well.

**Comment by yash-rajpal** [comment]
Ohh Okay, thanks for letting me know @singh-ravi-siso I will try to solve them too in this pr. Thanks. :)

**Comment by singh-ravi-siso** [comment]
@yash-rajpal  do read the issues properly, I have clearly written that I am working on it :)

**Comment by yash-rajpal** [comment]
Yeah, I read the issue properly. Last I remember, we weren't allowed to reserve issue.
Also I waited for sometime before making a pr, as you didn't, I solved the issue myself.

**Comment by singh-ravi-siso** [comment]
Okay. 
Please fix it in other modules as well, I will close my PR :)
